api/handlers/slack.ts

```diff
@@ -24,14 +24,19 @@ export async function buildEntitledBranchList(entitlement: any, repoBranchesRepo
     const [repoOwner, repoName, directoryPath] = repo.split('/');
     const branches = await repoBranchesRepository.getRepoBranches(repoName, directoryPath);
     for (const branch of branches) {
-      let buildWithSnooty = true;
       if ('buildsWithSnooty' in branch) {
-        buildWithSnooty = branch['buildsWithSnooty'];
-      }
-      if (buildWithSnooty) {
-        entitledBranches.push(
-          `${repoOwner}/${repoName}${directoryPath ? '/' + directoryPath : ''}/${branch['gitBranchName']}`
-        );
+        const buildWithSnooty = branch['buildsWithSnooty'];
+        if (buildWithSnooty) {
+          const active = branch['active'];
+          const repoPath = `${repoOwner}/${repoName}${directoryPath ? '/' + directoryPath : ''}/${
+            branch['gitBranchName']
+          }`;
+          if (!active) {
+            entitledBranches.push(`!Inactive ` + `${repoPath}`);
```

Comment:
total nit: Because of the font-style of Slack, I think it might be more readable with a lower-case `i`

Reply:
And up for debate - I'm wondering how to make it the most pleasing when we have no access to style. I wonder if it would look better in parentheses? 
`(!inactive) 10gen/docs/master`
wdyt?

Reply:
Oh another nit: since you're already using template strings, you can get rid of the concatenation!

`(!inactive) ${repoPath}`

Reply:
I definitely think in parentheses would look better! I was following the ticket exactly but thats a great idea, thank you; I'll confirm in the channel